pyCardDeck/deck.py

```diff
@@ -263,6 +266,77 @@ def show_top(self, number: int) -> List[CardType]:
         """
         return self._cards[0:number]
 
+    def set_file_location(self, location):
+        """
+        used to update the location
+        it will expand relative file path and ~ if it exists
+
+        :param location: the location of the file
```

Comment:
I assume that the location should be provided relative from the main script and should include filename? It'd be useful to document that